static/app/views/insights/common/components/widgets/cacheMissRateChartWidget.tsx

```diff
@@ -21,10 +21,17 @@ export default function CacheMissRateChartWidget(props: LoadableChartWidgetProps
     props.pageFilters
   );
 
+  // explore/alerts doesn't support `cache_miss_rate`, so this is used as a comparable query
+  const queryInfo = {
+    yAxis: [`${COUNT}()`],
```

Comment:
Should this be `count(span.duration)` when it translates over to explore/alerts? I think it pushes that value under the hood

Reply:
Hmm yeah I think you're right

Reply:
updated!